Python/pywarpx/EB.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 Lorenzo Giacomel
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+from .Bucket import Bucket
+
+eb = Bucket('eb2')
```

Comment:
One more comment. For clarity, it would be better if the object had the same name as the input group name. This should be `eb2 = Bucket('eb2')`.

Reply:
Sure! It's done now :)